definitions/ext-service/definition.yml

```diff
@@ -8,3 +8,9 @@ synthesis:
 
   conditions:
     - attribute: service.name
+
+  tags:
```

Comment:
@fryckbos you're expecting these to be aliased to k8s.clusterName etc. right?

We don't have this supported in the definition files yet (it is in the backend) but I will add a task to add it if that's the case.

Reply:
We've added support for this, you can add it like [this](https://github.com/newrelic-experimental/entity-synthesis-definitions#tags):

```
tags:
    k8s.cluster.name:
      entityTagName: k8s.clusterName # preferredTagName
```